client/gutenberg/extensions/contact-info/edit.js

```diff
@@ -0,0 +1,63 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { InnerBlocks } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+const ALLOWED_BLOCKS = [
+	'jetpack/markdown',
+	'jetpack/address',
+	'jetpack/email',
+	'jetpack/phone',
+	'jetpack/map',
+	'core/paragraph',
+	'core/image',
+	'core/heading',
+	'core/gallery',
+	'core/list',
+	'core/quote',
+	'core/shortcode',
+	'core/audio',
+	'core/code',
+	'core/cover',
+	'core/html',
+	'core/separator',
```

Comment:
Is `core/separator` the page separator and does it make sense to allow it here?

Reply:
I think so. What id you want to separate add a horizontal line in between the different address and the phone number?

Reply:
Oh it's `hr`? I was thinking it might be the thing that splits pages to 1, 2 3 etc subpages.

I totally agree to keep it if it's `<hr/>`